### PR TITLE
docs: improve README and remove copyright

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: {{cookiecutter.project_shortname}}/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| ``Dockerfile`` | Dockerfile used to build your application image. |
+| ``Pipfile`` | Python requirements installed via [pipenv](https://pipenv.pypa.io) |
+| ``Pipfile.lock`` | Locked requirements (generated on first install). |
+| ``app_data`` | Application data such as vocabularies. |
+| ``assets`` | Web assets (CSS, JavaScript, LESS, JSX templates) used in the Webpack build. |
+| ``docker`` | Example configuration for NGINX and uWSGI. |
+| ``docker-compose.full.yml`` | Example of a full infrastructure stack. |
+| ``docker-compose.yml`` | Backend services needed for local development. |
+| ``docker-services.yml`` | Common services for the Docker Compose files. |
+| ``invenio.cfg`` | The Invenio application configuratio. |
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/README.md:40]:
```suggestion
| ``invenio.cfg`` | The Invenio application configuration. |
```

File: {{cookiecutter.project_shortname}}/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+| ``logs`` | Log files. |
+| ``static`` | Static files that need to be served as-is (e.g. images). |
+| ``templates`` | Folder for your Jinja templates. |
+| ``.invenio`` | File used by Invenio-CLI. |
+| ``.invenio.private`` | File used by Invenio-CLI. |
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/README.md:45]:
```suggestion
| ``.invenio`` | Common file used by Invenio-CLI to be version controlled. |
| ``.invenio.private`` | Private file used by Invenio-CLI *not* to be version controlled. |
```

File: {{cookiecutter.project_shortname}}/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Documentation
+
+Do you want to learn how to configure, customize, deploy and much more. Visit
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/README.md:49]:
```suggestion
To learn how to configure, customize, deploy and much more, visit
```

File: {{cookiecutter.project_shortname}}/docker-compose.full.yml [hunk 4]
```diff
@@ -1,18 +1,12 @@
...
-# This file is part of Invenio.
-#
-# Copyright (C) 2019 CERN.
-# Copyright (C) 2019 Northwestern University.
-#
-# Invenio is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details.
-# Backend services needed for production.
+# Note, this file is not suitable for a production deployment. It is only an
+# example of all the infrastructure components needed a real production
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/docker-compose.full.yml:4]:
```suggestion
# example of all the infrastructure components needed in a real production
```

File: {{cookiecutter.project_shortname}}/docker-compose.yml [hunk 5]
```diff
@@ -1,19 +1,11 @@
...
-# Copyright (C) 2019 CERN.
-# Copyright (C) 2019 Northwestern University.
-#
-# Invenio is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details.
-#
 # Backend services needed for development.
 #
+# This file will start all the backend services needed to run InvenioRDM in
+# locally in development mode.
```
Comment (fenekku) [inline {{cookiecutter.project_shortname}}/docker-compose.yml:4]:
```suggestion
# This file will start all the backend services needed to run InvenioRDM
# locally in development mode.
```

File: {{cookiecutter.project_shortname}}/README.md (hunk 3 above)
Comment (ppanero) [reply {{cookiecutter.project_shortname}}/README.md]: If it's a question shouldn't it be:

`Do you want to learn how to configure, customize, deploy and much more?`

Anyway, just a super-ultra-minor :)

**Comment by lnielsen** [comment]
Fixed comments and merged as part of #85

**Pull request closed without merging**